fastchat/conversation.py

```diff
@@ -53,6 +53,7 @@ class Conversation:
     sep_style: SeparatorStyle = SeparatorStyle.ADD_COLON_SINGLE
     sep: str = "\n"
     sep2: str = None
+    prefix_token = ""
```

Comment:
This one seems to be only used for training. We want to keep this class minimal.
Could you avoid the modification of `conversation.py` and move this logit to your `train_with_template.py`?

Reply:
OK.

Reply:
I have reversed the changes in `class Conversation`